data/tlg0007/tlg081/tlg0007.tlg081.perseus-eng4.xml

```diff
@@ -0,0 +1,2182 @@
+		<fileDesc>
+			<titleStmt>
+				<title type="work">The apophthegms or remarkable sayings of kings and great
+					commanders</title>
+				<author>Plutarch</author>
+				<editor role="editor">Goodwin</editor>
+				<editor role="translator">Edward Hinton</editor>
+				<respStmt>
+					<resp>Prepared under the supervision of</resp>
+					<name>Lisa Cerrato</name>
+					<name>Rashmi Singhal</name>
+					<name>Bridget Almas</name>
+				</respStmt>
+				<funder n="org:NEH">The National Endowment for the Humanities</funder>
+			</titleStmt>
+			<publicationStmt>
+				<publisher>Trustees of Tufts University</publisher>
+				<pubPlace>Medford, MA</pubPlace>
+				<authority>Perseus Project</authority>
+				<date type="release">2010-12-13</date>
+			</publicationStmt>
+			<sourceDesc>
+				<biblStruct>
+					<monogr>
+						<author>Plutarch</author>
+						<title>Plutarch's Morals.</title>
+						<respStmt>
+							<resp>Translated from the Greek by several hands. Corrected and revised
+								by</resp>
+							<name>William W. Goodwin, PH. D.</name>
+						</respStmt>
+						<imprint>
+							<pubPlace>Boston</pubPlace>
+							<publisher>Little, Brown, and Company</publisher>
+							<pubPlace>Cambridge</pubPlace>
+							<publisher>Press Of John Wilson and son</publisher>
+							<date>1874</date>
+						</imprint>
+						<biblScope unit="volume">1</biblScope>
+					</monogr>
+				</biblStruct>
+			</sourceDesc>
+		</fileDesc>
+		<encodingDesc>
+			<editorialDecl>
+				<correction status="high" method="silent">
+					<p>optical character recognition</p>
+				</correction>
+			</editorialDecl>
+			<p>Text encoded in accordance with the latest EpiDoc standards</p>
+			<p>The following text is encoded in accordance with EpiDoc standards and with the
+				CTS/CITE Architecture</p>
+			<refsDecl n="CTS">
+				<cRefPattern n="section" matchPattern="(\w+)"
+					replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+					<p>This pointer pattern extracts sections</p>
+				</cRefPattern>
+				<!-- <state unit="section"/> -->
+			</refsDecl>
+			<!--<refsDecl doctype="TEI.2">
+				<state unit="stephpage" n="chunk"/>
+				</refsDecl>-->
+		</encodingDesc>
+		<profileDesc>
+			<langUsage>
+				<language ident="eng">English</language>
+				<language ident="grc">Greek</language>
+				<language ident="lat">Latin</language>
+			</langUsage>
+		</profileDesc>
+		<revisionDesc>
+			<change who="GRC" when="2006">tagging</change>
+			<change who="1ucyP" when="2019-06-28">EpiDoc and CTS Conversion</change>
+		</revisionDesc>
+	</teiHeader>
+	<text xml:lang="eng">
+		<body>
+			<div type="edition" xml:lang="eng" n="urn:cts:greekLit:tlg0007.tlg081.perseus-eng4">
+				<pb xml:id="v.1.p.185"/>
+				<head>The apophthegms or remarkable sayings of kings and great commanders.</head>
+				<div type="textpart" subtype="section" n="1">
+					<head>ARTAXERXES</head>
+					<p>ARTAXERXES, King of Persia, O Caesar Trajan, greatest of princes, esteemed it
```

Comment:
Hmmm, we have duplicate names here. The names appear inline with the text in the print edition, so these headers are not found in the print.
I would suggest changing to `<head resp="perseus">
I'm not sure I like them here, but I can see where it makes alignment with the eng3 file easier.